src/vl-rich-data-table.js

```diff
@@ -54,31 +55,42 @@ export class VlRichDataTable extends VlElement(HTMLElement) {
                 @import "/node_modules/vl-ui-icon/dist/style.css";
                 @import "/node_modules/vl-ui-button/dist/style.css";
                 @import "/node_modules/vl-ui-data-table/dist/style.css";
+                @import "/node_modules/vl-ui-modal/dist/style.css";
             </style>
-            <div is="vl-grid" is-stacked>
-                <div is="vl-column" id="toggle-filter" class="vl-u-align-right" hidden size="12">
-                    <button id="toggle-filter-button" is="vl-button-link" type="button" aria-label="Toon de filter">
-                        <span is="vl-icon" icon="content-filter" before></span><slot name="toggle-filter-button-text">Filter</slot>
-                    </button>
-                </div>
-                <div id="search" is="vl-column" size="0">
-                    <button id="close-filter-button" class="vl-filter__close" hidden type="button">
-                        <span is="vl-icon" icon="close"></span>
-                        <span class="vl-u-visually-hidden"><slot name="close-filter-button-text">Filter sluiten</slot></span>
-                    </button>
-                    <slot name="filter"></slot>
-                </div>
-                <div id="content" is="vl-column" size="12">
-                    <table is="vl-data-table">
-                        <thead>
-                            <tr></tr>
-                        </thead>
-                        <tbody></tbody>
-                    </table>
-                </div>
-                <div id="pager" is="vl-column" size="12">
-                    <slot name="pager"></slot>
+            <div>
+                <div is="vl-grid" is-stacked>
+                    <div id="toggle-filter" is="vl-column" class="vl-u-align-right vl-u-hidden--s" hidden size="12">
+                        <button id="toggle-filter-button" class="" is="vl-button-link" type="button" aria-label="Toon de filter">
```

Comment:
lege class?

Reply:
Ik vermoed dat de opmerking gaat over de nieuwe container div? Ik denk dat de modal ook gewoon in de vl-grid gedeclareerd kan worden.

Reply:
Nee het leek te gaan over `class=""` . De modal moest buiten de vl-grid omdat anders wat stijl werd overgenomen die niet echt wenselijk was.